logos/tests/performance/run_eccs_ablation.sh

```
@@ -0,0 +1,180 @@
+    echo "  Run 2: ECCS DISABLED"
+    echo "────────────────────────────────────────────────────────────────"
+
+    ECCS_OFF_ENV="ECCS_ETTFT_ENABLED=false"
+    if [[ -n "$WEIGHT_OVERRIDE" ]]; then
+        ECCS_OFF_ENV="$ECCS_OFF_ENV ECCS_WEIGHT_OVERRIDE=$WEIGHT_OVERRIDE"
+    fi
+
+    docker compose exec \
+        -e "ECCS_DECISION_LOG=/tmp/eccs_decisions_off.jsonl" \
+        -e "ECCS_ETTFT_ENABLED=false" \
+        ${WEIGHT_OVERRIDE:+-e "ECCS_WEIGHT_OVERRIDE=$WEIGHT_OVERRIDE"} \
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes in this variable will not be respected.

(SC2090)